src/adapter/src/coord/appends.rs

```diff
@@ -363,21 +289,24 @@ impl<S: Append + 'static> Coordinator<S> {
 
     /// Enqueue requests to advance all local inputs (tables) to the current wall
     /// clock or at least a time greater than any previous table read (if wall
-    /// clock has gone backward). These are not processed in a single append call
-    /// because they currently processed serially by persist. In order to allow
-    /// other coordinator messages to be processed (like user queries), split up the
-    /// processing of this work.
+    /// clock has gone backward).
     pub(crate) async fn queue_local_input_advances(&mut self, advance_to: mz_repr::Timestamp) {
```

Comment:
This name may no longer be totally accurate.

Reply:
I think it is! It enqueues work to be done!